setup.py

```diff
@@ -30,11 +30,12 @@
     "markdown>=2.6.11",
     "numpy==1.14.3",
     "passlib==1.7.1",
-    "plotly==2.0.15",
+    "plotly>3.0.0",
     "pyyaml==3.12",
     "SQLAlchemy>=1.1.5",
     "Werkzeug==0.14.1",
     "WTForms>=2.1",
+    "dash @ https://github.com/TMiguelT/dash/archive/master.zip#sha1=e47990eec8fa3f9069040db86ad6eb9a2ad77ce2",
```

Comment:
I don't think that this is being handled properly when I run `python setup.py develop`:

```
Searching for dash@ https://github.com/TMiguelT/dash/archive/master.zip#sha1=e47990eec8fa3f9069040db86ad6eb9a2ad77ce2
Reading https://pypi.python.org/simple/dash/
Downloading https://files.pythonhosted.org/packages/e1/28/c1669c7173cb4b6df44a39122d0e6667cb7eb4a42ca9fa4c3ebd5b90dced/dash-1.0.0.tar.gz#sha256=a5abcf9df0641d8544f2dcd52829891bf2001f697921452b7084694e49b71128
Best match: dash 1.0.0
Processing dash-1.0.0.tar.gz
```